rep-2000.rst

```diff
@@ -0,0 +1,121 @@
+REP: 2000
+Title: ROS 2.0 Target Platforms
+Author: Mikael Arguedas <mikael@openrobotics.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 10-Apr-2018
+
+
+Abstract
+========
+
+This REP defines target platforms for each ROS 2.0 Distribution Release.
+We define platforms to include both operating system releases (e.g. Ubuntu
+Xenial (16.04 LTS)) as well as major language releases (e.g. Python 3.5). The
+target platforms represent the set on which all core stacks are
+expected to work. Exceptions can be made for stacks that are
+intentionally platform-specific.
+
+Platforms by Distribution
+=========================
+
+If planned support for a target platform is changed, notice will be
+sent to ros-users to enable discussion of this issue.
+
+Ardent Apalone (December 2017 - December 2018)
+----------------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- MacOS El Capitan (10.11)
+- MacOS Sierra (10.12)
+- MacOS High Sierra (10.13)
+- Windows 10
+
+Recommended Support for:
+
+- Debian Stretch (9)
+
+Architectures Supported:
+
+- amd64
+- arm64
+
+Targeted Languages:
+
+- C++14
+- Python 3.5
+
+Requirements:
+
+
++---------+---------------+---------------+----------------+
+| Package | Ubuntu Xenial |     MacOS**   |   Windows 10** |
++=========+===============+===============+================+
+| CMake   |     3.5.1     |     3.11.0    |     3.10.2     |
++---------+---------------+---------------+----------------+
+| EmPY    |     3.3.2     |     3.6.5     |     3.3.2      |
++---------+---------------+---------------+----------------+
+| Poco    |     1.7.7     |     1.7.7     |     1.7.7      |
```

Comment:
Can we merge the cells if the value is the same across all columns? That would be easier to see / read.